src/models/wallet/SimpleWallet.ts

```diff
@@ -141,7 +151,7 @@ export class SimpleWallet extends Wallet {
     }
     return new SimpleWallet(
       wallet.name,
-      wallet.network,
+      Number(wallet.network),
```

Comment:
I'm hesitant to accept this change without an associated unit test. All the new SDK code that team NEM writes is via TDD. I think we need to do that as well.

Reply:
Also probably makes sense to run the existing unit tests and see what fails